packages/react-composites/src/composites/CallWithChatComposite/styles/MobilePane.styles.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, IButtonStyles, IStackStyles } from '@fluentui/react';
+
+export const mobilePaneStyle = { root: { width: '100%', height: '100%' } };
+
+export const hiddenMobilePaneStyle: IStackStyles = concatStyleSets(mobilePaneStyle, { root: { display: 'none' } });
+
+export const mobilePaneControlBarStyle: IStackStyles = { root: { height: '3rem' } };
+
+export const mobilePaneBackButtonStyles: IButtonStyles = {
+  root: { border: 'none', minWidth: '2.5rem', height: '100%', background: 'none', padding: '0 1rem 0 0.5rem' },
+  rootChecked: { background: 'none' },
+  rootCheckedHovered: { background: 'none' }
+};
+
+export const mobilePaneButtonStyles: IButtonStyles = {
+  root: {
+    border: 'none',
+    borderBottom: '0.125rem solid transparent',
+    minWidth: '8rem',
+    maxWidth: '8rem',
+    borderRadius: 'none',
```

Comment:
Why do you have to explicitly set the `borderRadius`? This should be the default.

Reply:
It's not needed. Removed. Thanks.